.github/workflows/ci.yml

```diff
@@ -37,20 +38,26 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-py${{ matrix.python }}-
       - name: Install dev dependencies
-        run: pip install --requirement requirements-dev.txt
+        run: >
+          pip install
+          --requirement requirements-dev.txt
```

Comment:
What's the reason for making this and other changes below multi line like this?

Reply:
i started by wrapping everything, and then selectively trying to remove as many arguments and configs as possible.

turned out i could actually remove pretty much everything, and left it there.

but apparently others care about this so i put the short commands back on a single line just now 🙃